app/lib/shared/configuration.dart

```diff
@@ -131,6 +131,13 @@ class Configuration {
       throw Exception('Unknown project id: ${env.gcloudProject}');
     }
   }
+
+  /// Whether the current project is the production one.
+  bool get isProdProjectId => projectId == 'dartlang-pub';
+
+  /// The PUB_HOSTED_URL to use in analyzer and dartdoc.
+  String get pubHostedUrl =>
+      isProdProjectId ? null : 'https://dartlang-pub-dev.appspot.com';
```

Comment:
Can't we avoid this by always using `'https://${projectId}.appspot.com'` ? I don't see any downside to using that in production...

Reply:
We could also just make this a regular field... so that it's set depending on wether the `_prod()` or `_dev()`  constructor is used... It's simple and consistent with how all fields are set..

Also no reason we couldn't use `https://pub.dartlang.org` in production...

Reply:
Updated, PTAL.